src/plugins/data/public/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -153,11 +153,17 @@ export class IndexPattern implements IIndexPattern {
     if (!this.id) {
       return;
     }
-
-    if (forceFieldRefresh || this.isFieldRefreshRequired()) {
-      await this.refreshFields();
+    try {
+      if (forceFieldRefresh || this.isFieldRefreshRequired()) {
+        await this.refreshFields();
+      }
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(`Refreshing fields of index pattern ${this.id} failed`, {
+        forceFieldRefresh,
+        fields: this.fields,
+      });
```

Comment:
So this is the mystery, normally, in discover, `forceFieldRefresh` is false, and also `this.isFieldRefreshRequired` should also return false. Currently I just could reproduce it with setting `forceFieldRefresh` manually to `true`. So the UI in Discover and Management works again. The question is, what causes the `this.isFieldRefreshRequired`  to be true?